be/src/exec/pipeline/scan/scan_operator.cpp

```diff
@@ -161,16 +167,12 @@ StatusOr<vectorized::ChunkPtr> ScanOperator::pull_chunk(RuntimeState* state) {
         _workgroup->incr_period_ask_chunk_num(1);
     }
 
-    for (auto& chunk_source : _chunk_sources) {
-        if (chunk_source != nullptr && chunk_source->has_output()) {
-            auto&& chunk = chunk_source->get_next_chunk_from_buffer();
-            eval_runtime_bloom_filters(chunk.value().get());
-
-            return std::move(chunk);
-        }
+    vectorized::ChunkPtr res = get_chunk_from_buffer();
```

Comment:
`ConnectorScanOperator` doesn't implement `get_chunk_from_buffer()`, which will always returns `nullptr`.

Reply:
Good advice, I should override the `ConnectorScanOperator::get_chunk_from_buffer`.